blog-samples/CSharp/FacebookHandover/FacebookModel/FacebookPage.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace FacebookModel
+{
+    /// <summary>
+    /// Defines a Facebook sender.
```

Comment:
This comment might be better as 'Defines a Facebook Id', since it is not always the id of the sender.